src/GovUk.Education.ExploreEducationStatistics.Data.Api/appsettings.json

```diff
@@ -2,6 +2,11 @@
   "Logging": {
     "LogLevel": {
       "Default": "Information"
+    },
+    "ApplicationInsights": {
```

Comment:
Data API doesn't have this line which Admin has. Does it need it?
`builder.AddApplicationInsights(Configuration.GetSection("AppInsights").GetValue<string>("InstrumentationKey"));`

If not, what's the difference?

Reply:
This is covered under a separate piece of work reviewing how app insights should be configured